### ISSUE TITLE
Standard Height for Staff Cards + Icon Spacing for 6 Social Icons

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1499]
Not all staff cards will include the same amount of information, example: https://develop--clarkcountynv--aemsites.aem.page/drafts/sarnold/accessibility-release-content-setup/contact-us

1. Need to standardize the height of the cards in a block OR at least in a row so that they are consistent heights. 

https://develop--clarkcountynv--aemsites.aem.page/drafts/sarnold/accessibility-release-content-setup/county-manager/

1. Need to adjust spacing of icons slightly for use case with 6 icons
2. Youtube logo appearing as second facebook logo

Acceptance Criteria: 
1. When viewing staff cards, when cards have different amounts/length of content, then the cards in a row are consistent height. 
2. When viewing staff cards, when 6 social media icons are in the footer of the cards, then there should be appropriate spacing for icons.
3. When viewing staff cards, when a youtube link is in footer, then it should display youtube logo.
